### PR TITLE
fix external links, tab-icon mess, #270

### PR DESCRIPTION
**Pull request by patriknw**
References #270

### CODE REVIEW

File: docs-source/docs/modules/concepts/pages/akka-cluster.adoc [hunk 1]
```diff
@@ -31,8 +31,8 @@ NOTE: The xref:microservices-tutorial:entity.adoc[Event Sourced Cart entity in t
 
 When operating an Akka cluster you must consider how to handle network partitions (a.k.a. split brain scenarios) and machine crashes (including JVM and hardware failures). This is crucial for correct behavior if you use Cluster Singleton or Cluster Sharding, especially together with xref:akka-persistence.adoc[].
 
-The {akka-blog}/news/2020/06/08/akka-split-brain-resolver-video[Split Brain Resolver video] {akka-blog}/news/2020/06/08/akka-split-brain-resolver-video[{tab-icon}, window="tab"] is a good starting point for learning why it is important to use a correct downing provider and how the Split Brain Resolver works.
+The {akka-blog}/news/2020/06/08/akka-split-brain-resolver-video[Split Brain Resolver video  {tab-icon}, window="tab"] is a good starting point for learning why it is important to use a correct downing provider and how the Split Brain Resolver works.
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/akka-cluster.adoc:34]:
```suggestion
The {akka-blog}/news/2020/06/08/akka-split-brain-resolver-video[Split Brain Resolver video {tab-icon}, window="tab"] is a good starting point for learning why it is important to use a correct downing provider and how the Split Brain Resolver works.
```

File: docs-source/docs/modules/concepts/pages/message-driven-event-driven.adoc [hunk 2]
```diff
@@ -17,17 +17,17 @@ An Event is some data emitted from a component for anyone listening to consume.
...
 
 == Events _are_ messages
 
 As discussed above, in Message Driven systems, each component send items to a fixed recipient. In Event Driven systems, on the other hand, each component produces items of data with a fixed sender and shares them with any consumer. This comparison, though, can be misleading: the term 'Message Driven' refers to a building block on a system and 'Event Driven' refers to a higher level property of a system. So, using Message Driven tools we can build an Event Driven system.
 
-There are multiple types of messages. A categorization of messages in a xref:cqrs.adoc[CQRS]/xref:event-sourcing[ES] application is the following:
+There are multiple types of messages. A categorization of messages in a xref:cqrs.adoc[CQRS]/xref:event-sourcing.adoc[ES] application is the following:
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/message-driven-event-driven.adoc:28]: Nice catch!

File: docs-source/docs/modules/concepts/pages/reactive-disambiguation.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ Introduced in 2013, the Reactive Manifesto summarizes the properties a system mu
 * *Elastic*: The system stays responsive under varying workload.
 * *Message Driven*: Reactive Systems rely on asynchronous message-passing to establish a boundary between components that ensures loose coupling, isolation and location transparency.
 
-Read the whole manifesto at {reactive-manifesto}[] {reactive-manifesto}[{tab-icon}, window="tab"].
+Read the whole manifesto at {reactive-manifesto}[https://www.reactivemanifesto.org/{tab-icon}, window="tab"].
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/reactive-disambiguation.adoc:15]:
```suggestion
Read the whole manifesto at {reactive-manifesto}[https://www.reactivemanifesto.org/ {tab-icon}, window="tab"].
```

File: docs-source/docs/modules/concepts/pages/reactive-disambiguation.adoc [hunk 4]
```diff
@@ -33,7 +33,7 @@ The eight foundational principles of implementing a distributed system that make
 . Handle Dynamics
 
 
-Read all the details (and the patterns) at {reactive-principles}[] {reactive-principles}[{tab-icon}, window="tab"]  .
+Read all the details (and the patterns) at {reactive-principles}[https://principles.reactive.foundation  {tab-icon}, window="tab"].
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/reactive-disambiguation.adoc:36]:
```suggestion
Read all the details (and the patterns) at {reactive-principles}[https://principles.reactive.foundation {tab-icon}, window="tab"].
```

File: docs-source/docs/modules/how-to/pages/scale-independently.adoc [hunk 5]
```diff
@@ -167,6 +167,6 @@ Or, using the JVM arguments syntax:
 
 == Learn more
 
-* {akka}/typed/cluster.html#node-roles[Akka Cluster Node Roles] {akka}/typed/cluster.html#node-roles[{tab-icon}, window="tab"]
-*  {akka}/typed/cluster-sharding.html[Akka Cluster Sharding] {akka}/typed/cluster-sharding.html[{tab-icon}, window="tab"]
-* {akka}/typed/cluster-sharded-daemon-process.html#sharded-daemon-process[Sharded Daemon Process] {akka}/typed/cluster-sharded-daemon-process.html#sharded-daemon-process[{tab-icon}, window="tab"]
+* {akka}/typed/cluster.html#node-roles[Akka Cluster Node Roles {tab-icon}, window="tab"]
+*  {akka}/typed/cluster-sharding.html[Akka Cluster Sharding {tab-icon}, window="tab"]
```
Comment (ignasi35) [inline docs-source/docs/modules/how-to/pages/scale-independently.adoc:171]:
```suggestion
* {akka}/typed/cluster-sharding.html[Akka Cluster Sharding {tab-icon}, window="tab"]
```

This is unrelated but a fix anyway 😅

File: docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc [hunk 6]
```diff
@@ -6,7 +6,7 @@ include::ROOT:partial$include.adoc[]
...
 
-This tutorial uses Cassandra and Kafka. You may run local installations during development, but we recommend running both services using https://docs.docker.com[Docker {tab-icon}, window="tab"]. The template project's root directory includes a https://docs.docker.com/compose/[Docker Compose] https://docs.docker.com/compose/[{tab-icon}, window="tab"] file that you can use to run Cassandra and Kafka. The provided Docker Compose file also runs Zookeeper which is required to run a Kafka broker.
+This tutorial uses Cassandra and Kafka. You may run local installations during development, but we recommend running both services using https://docs.docker.com[Docker {tab-icon}, window="tab"]. The template project's root directory includes a https://docs.docker.com/compose/[Docker Compose {tab-icon}, window="tab"]] file that you can use to run Cassandra and Kafka. The provided Docker Compose file also runs Zookeeper which is required to run a Kafka broker.
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/dev-env.adoc:9]:
```suggestion
This tutorial uses Cassandra and Kafka. You may run local installations during development, but we recommend running both services using https://docs.docker.com[Docker {tab-icon}, window="tab"]. The template project's root directory includes a https://docs.docker.com/compose/[Docker Compose {tab-icon}, window="tab"] file that you can use to run Cassandra and Kafka. The provided Docker Compose file also runs Zookeeper which is required to run a Kafka broker.
```

**Review by ignasi35 — APPROVED** [review]
This is great! Thanks @patriknw 

I've left a handful of minor suggestions (double spaces and other syntax errors)

File: docs-source/docs/modules/concepts/pages/message-driven-event-driven.adoc (hunk 2 above)
Comment (patriknw) [reply docs-source/docs/modules/concepts/pages/message-driven-event-driven.adoc:28]: yeah, I don't know when it is needed and not. IntelliJ warned about this one, but I have seen others without.

**Comment by patriknw** [comment]
thanks for fixing the additional mistakes, Ignasi

**Review by octonato — APPROVED** [review]

**Pull request merged**